.github/workflows/ci.yml

```diff
@@ -183,7 +183,7 @@ jobs:
       run: cargo test -p winit-core
 
     - name: Test winit Android
-      if: contains(matrix.platform.target, 'android')
+      if: contains(matrix.platform.target, 'android') && false # TODO: Fix this
```

Comment:
Well, I think that should be addressed somehow though? Maybe the issue is that we're running tests with `-p`, though, doubt it...

Reply:
Added `--no-run`, I think that should fix it